code/modules/mob/living/carbon/human/unarmed_attack.dm

```diff
@@ -97,10 +97,10 @@ var/global/list/sparring_attack_cache = list()
 //Factor in attackspeed here
 /datum/unarmed_attack/proc/get_delay(var/mob/living/user)
 	if (isnum(delay) && delay > 0)
-		.= (delay / user.get_attack_speed_factor())
+		var/real_delay = delay / user.get_attack_speed_factor()
```

Comment:
any reason for this change?

Reply:
Nvm, that was part of old changes because instead of `. =` we've used `delay /=`